src/Waives.Extensions.DocumentChannels.Filesystem/FileSystemDocumentEmitter.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Waives.Reactive;
+
+namespace Waives.Extensions.DocumentChannels.Filesystem
+{
+    public sealed class FileSystemDocumentEmitter : DocumentEmitter, IDisposable
```

Comment:
✋ Needs some xmldoc comments explaining what the class does given this is `public`

Reply:
✅ in e12ccc1